### PR TITLE
Add documentation for 0.7.1-alpha release.

### PR DESCRIPTION
**Pull request by kowshikn**
*Issue #288*
*Description of changes:*

Add some documentation and included the Cfn from the master. 

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: docs/readme/trace_overview.md [hunk 1]
```diff
@@ -69,3 +86,13 @@ service-map-pipeline:
         password: "admin"
         trace_analytics_service_map: true
 ```
+## Setup
+As mentioned in the above we have a specific [pipeline.yml](trace_overview.md#trace-analytics-pipeline-configuration) setup for achieving Trace Analytics in Data Prepper. Below are more detailed links of launching Data Prepper.
```
Comment (kowshikn) [inline docs/readme/trace_overview.md:90]: nit: Drop the `in the`

File: deployment/aws/README.md [hunk 2]
```diff
@@ -0,0 +1 @@
+Work In Progress
```
Comment (kowshikn) [inline deployment/aws/README.md:1]: @wrijeff I am going to let you fill this. Changed the structure, `deployment/aws/ec2`

File: data-prepper-plugins/elasticsearch/security.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
+# Elasticsearch Sink Security
+
+This document provides more details about the security settings of the sink.
+
+## AWS Elasticsearch Service
+
+Elasticsearch sink is capable of sending data to Amazon Elasticsearch domain that uses Identity and Access Management. The plugin uses the default credential chain. Run `aws configure` using the AWS CLI to set your credentials.
```
Comment (wrijeff) [inline data-prepper-plugins/elasticsearch/security.md:7]: domains which use*

Also users don't need `aws configure` if they're running on EC2 w/ a service role attached. Might be worth calling it out somewhere in this doc (I can add it as a follow up, need to add to the deployment readme anyway).

File: data-prepper-plugins/elasticsearch/security.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+        "AWS": "arn:aws:iam::<AccountId>:user/data-prepper-sink-user"
+      },
+      "Action": "es:ESHttpGet",
+      "Resource": "arn:aws:es:us-east-1:<AccountId>:domain/<domain-name>/_cluster/settings"
+    }
+  ]
+}
+``` 
+
+Please check this [doc](https://docs.aws.amazon.com/elasticsearch-service/latest/developerguide/es-ac.html) to know how to set IAM to your elasticsearch domain,
```
Comment (wrijeff) [inline data-prepper-plugins/elasticsearch/security.md:40]: Capitalized Elasticsearch (comment applies throughout)

File: docs/readme/trace_overview.md [hunk 5]
```diff
@@ -69,3 +86,13 @@ service-map-pipeline:
...
+
+##Open distro For Elasticsearch
+
+Please check this [link](https://opendistro.github.io/for-elasticsearch-docs/docs/trace/data-prepper/)  for launching Data Prepper to send data to Open Distro For Elasticsearch docs. 
+
+##AWS Elasticsearch
+
+Please check this [link](../../deployment/aws/README.md)  for launching Data Prepper to send data to AWS Elasticseaech service.
```
Comment (wrijeff) [inline docs/readme/trace_overview.md:98]: Typo in Elasticsearch

**Review by wrijeff — APPROVED** [review]
👍 all minor comments. I can address in my follow-up PR to add the deployment readme

**Pull request merged**